### PR TITLE
Associations and Advanced Querying: Private Events!

### PR DESCRIPTION
**Pull request by lmaldonadoch**
On this project, we learned about Advanced Querying and Advanced Associations. We were able to apply all the previously acquired knowledge plus some new knowledge on how to retrieve data from Databases in an efficient way and how to make associations for many-to-many relationships.

## Project milestones
- Create a User Model, Controler, and Views.
- Create a simple sign-in function for Users.
- Build Events Model, Controller, and Views.
- Make the appropriate Associations for Users and Events.
- Make it possible for users to create events.
- Make it possible for users to attend events.
- Make it possible for users to show the previous and upcoming events per user.
- Make it possible for events to be displayed by previous and upcoming.

# Requirements

## Section:
- Do not commit debugging instructions - read discussions about its reasons here.
- Keep your code clean.
- Remember to not change migration files after they are committed (explanation why it is important).
- Make sure that changes in migrations files are reflected in DB/schema.rb (explanation why it is important).
- Make sure that there is no logic in views and that helpers are responsible only for view related logic.
   - Where Do I Put My Code?
   - The Beginner’s Guide to Rails Helpers

## General requirements:
- [x] Add a descriptive README file to your repositories as described here.
- [x] Use meaningful names for your Pull Requests and add a short summary of its content as described here.
- [x] Use descriptive commit messages as described here.
- [x] Do not commit files that are not related to the project. Instead, add them to .gitignore file.
- [x] Do not commit inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.

## Git Flow:
- [x] Use a specific branch for each feature.
- [x] Use a descriptive name for your branches.
- [x] Use Rubocop to validate your code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,98 @@
...
+```
+$ bundle install
+```
+
+Finally, migrate the database:
+
+```
+$ rails db:migrate
+```
+
```
Comment (mcrd25) [inline README.md:45]: [OPTIONAL]

You should add the command to run app here as well

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+class EventsController < ApplicationController
+  def new
+    @event = Event.new
+  end
+
+  def create
+    current_user = User.find(session[:current_user_id])
+    @event = current_user.events.build(event_params)
+    # @event.date = 'Tomorrow'
```
Comment (mcrd25) [inline app/controllers/events_controller.rb]: Please remove all commented out code 🙂 
As can be seen from [this screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
It is better to delete commented code. If you need to review a previous version of your code, use git history. 🙂

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# Changes Required

Hi Team!

Good job! You have most of the functionality down! Good job! 
You only have one minor change to address and yo can get the 👍 


***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

File: app/controllers/events_controller.rb (hunk 2 above)
Comment (shubham14p3) [reply app/controllers/events_controller.rb]: Thanks for the Feedback, @mcrd25,we have deleted the commented code as requested.

File: README.md (hunk 1 above)
Comment (shubham14p3) [reply README.md:45]: Run command added :)

**Review by Torres-ssf — APPROVED** [review]
### Project Approved :+1: :+1: :+1: 
Hi there team :hand: 
Great job implementing the requested changes.
**Your project it's approved**
Good luck in the next one.

Happy coding :computer: 
**Sergio Torres**

**Pull request merged**